### ISSUE TITLE
Client 2.4.3 errors out when executing a query with a hot cache

### ISSUE THREAD

**Issue by GlennMatthys** [apollographql/apollo-client#4034]
When a query runs for a second time (for example navigating back to an already visited page) and there's data in the cache, QueryManager crashes.

The data is fetched through vue-apollo. It uses a watch query, the fetchPolicy is set to 'cache-and-network'.

**Intended outcome:**
Query runs and fills cache as usual

**Actual outcome:**
Execution errors out:
```
QueryManager.js?96af:414 Uncaught TypeError: Invalid property descriptor. Cannot both specify accessors and a value or writable attribute, #<Object>
    at Function.defineProperty (<anonymous>)
    at handleKey (cloneDeep.js?1e56:28)
    at Array.forEach (<anonymous>)
    at cloneDeepHelper (cloneDeep.js?1e56:30)
    at eval (cloneDeep.js?1e56:13)
    at Array.forEach (<anonymous>)
    at cloneDeepHelper (cloneDeep.js?1e56:12)
    at handleKey (cloneDeep.js?1e56:27)
    at Array.forEach (<anonymous>)
    at cloneDeepHelper (cloneDeep.js?1e56:30)
```

![apollo-2 4 3-error](https://user-images.githubusercontent.com/31989842/47341683-f13a2b00-d6a1-11e8-9873-24c5d97b7aa0.png)

**How to reproduce the issue:**
Working on a jsfiddle

**Versions**
Fails with:
```
  System:
    OS: Windows 10
  Binaries:
    npm: 5.6.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 42.17134.1.0
  npmPackages:
    apollo-client: ^2.4.3 => 2.4.3
    vue-apollo: ^3.0.0-beta.11 => 3.0.0-beta.25
    vue-cli-plugin-apollo: ^0.17.2 => 0.17.2

```
Works with:
```
  System:
    OS: Windows 10
  Binaries:
    npm: 5.6.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 42.17134.1.0
  npmPackages:
    apollo-client: ^2.4.2 => 2.4.2
    vue-apollo: ^3.0.0-beta.11 => 3.0.0-beta.25
    vue-cli-plugin-apollo: ^0.17.2 => 0.17.2
```

**Comment by dongsuo** [comment]
Add this line to your package.json to temporarily solve this.
`"apollo-utilities": "1.0.20"`

**Comment by benjamn** [comment]
@GlennMatthys No, this is a new problem, which I accidentally introduced when I replaced that `fclone` package yesterday: https://github.com/apollographql/apollo-client/commit/78e2ad89f950da2829f49c7876f968adb2bc1302. Fix coming soon!

**Comment by benjamn** [comment]
In case there wasn't a GitHub notification, you should be able to fix this by running
```bash
npm install apollo-utilities@latest
```
to get version 1.0.23.

**Comment by frankdugan3** [comment]
@benjamn `apollo-client@2.4.3` still pulls in `apollo-utilities@1.0.22` as a nested dep, which I can override with yarn's resolutions, but even after I do that, queries work correctly but mutations still produce the same error.

**Comment by benjamn** [comment]
See also https://github.com/apollographql/apollo-client/pull/4039

**Comment by NicoAiko** [comment]
> In case there wasn't a GitHub notification, you should be able to fix this by running
> 
> ```shell
> npm install apollo-utilities@latest
> ```
> 
> to get version 1.0.23.

This fixed my problem, thanks!

**Comment by Akryum** [comment]
Version `1.0.23` doesn't fix the issue in vue-cli, but forcing `1.0.21` works.

**Comment by ghaddon** [comment]
I just started getting the same error. Is Apollo-utilities@latest for server or client?

**Comment by alx13** [comment]
I can confirm that 1.0.23 doesn't fix all `deepClone` issues.
Besides, apollo-client package is sticked to [explicit](https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/package.json#L54) version of apollo-utilities, so it doesn't get 1.0.23 until its republished:
```
"apollo-utilities": "1.0.22"
```

**Comment by GlennMatthys** [comment]
@benjamn This issue should be reopened as it doesn't appear to be fixed yet.

**Comment by benjamn** [comment]
This should now be fixed with `apollo-utilities@1.0.24` (just published).

**Comment by ghaddon** [comment]
Just tested with apollo-utilities@1.0.24 and is working for me. THX...
